repo-include.ks.in

```diff
@@ -0,0 +1,69 @@
+#test name: repo-include
+
+## Test for repo --cost and --includepkgs.
+
+%ksappend common/common_no_payload.ks
+%ksappend repos/default.ks
+
+# The addon repos are setup with the following packages:
```

Comment:
Thank you, this is perfect. I wonder if this explanation is the part missing in the other tests and hampering comprehension.